<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClient</class>
 <widget class="QWidget" name="CClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kundenstamm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MEGAMover/MEGAMover.qrc">
    <normaloff>:/icons/MM.png</normaloff>:/icons/MM.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblKdNr">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Kunden-Nr.:</string>
       </property>
       <property name="buddy">
        <cstring>grpTableControl</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtKdNr">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblMatch">
       <property name="text">
        <string>Match-Code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtMatch"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpAdress">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Adress-Daten</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="grpAG">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Auftraggeber</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="AdressPad" name="padAG" native="true"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="cmdNewAdrAG">
               <property name="text">
                <string>Neue Adresse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cmdSrchAdrAG">
               <property name="text">
                <string>Adresse auswählen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpRE">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Rechnungs-Anschrift</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="AdressPad" name="padRE" native="true"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="cmdNewAdrRE">
               <property name="text">
                <string>Neue Adresse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cmdSrchAdrRE">
               <property name="text">
                <string>Adresse auswählen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CClientAccountPad" name="padAccount" native="true">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Konto-Informationen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="grpTableControl" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Kunden-Auswahl: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdNewClient">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Neu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdPrevClient">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdSrchClient">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Suchen...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdNextClient">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdSaveClient">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Speichern des aktuellen Kundensatzes</string>
        </property>
        <property name="statusTip">
         <string>Speichern des aktuellen Kundensatzes</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../MEGAMover/MEGAMover.qrc">
          <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdDeleteClient">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Löschen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="grpWindowControl" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdClose">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Schließen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdressPad</class>
   <extends>QWidget</extends>
   <header>adresspad.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CClientAccountPad</class>
   <extends>QWidget</extends>
   <header>cclientaccountpad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtKdNr</tabstop>
  <tabstop>txtMatch</tabstop>
  <tabstop>cmdNewAdrAG</tabstop>
  <tabstop>cmdSrchAdrAG</tabstop>
  <tabstop>cmdNewAdrRE</tabstop>
  <tabstop>cmdSrchAdrRE</tabstop>
  <tabstop>cmdNewClient</tabstop>
  <tabstop>cmdPrevClient</tabstop>
  <tabstop>cmdSrchClient</tabstop>
  <tabstop>cmdNextClient</tabstop>
  <tabstop>cmdDeleteClient</tabstop>
  <tabstop>cmdClose</tabstop>
 </tabstops>
 <resources>
  <include location="../MEGAMover/MEGAMover.qrc"/>
 </resources>
 <connections/>
</ui>
